.github/workflows/zowe-cli-plugin.yml

```diff
@@ -40,11 +34,9 @@ jobs:
       run: npm install -g npm@7
 
     - name: Install Node Package Dependencies
-      id: install
       run: npm ci

```

Comment:
should we call the `zowe-actions/octorelease-script` `npmUpdate` script right after `npm ci` ?

Reply:
We probably should, and I am going to update it.